src/components/Icon/FontIcon/FontIcon.jsx

```diff
@@ -2,20 +2,25 @@ import React, { forwardRef } from "react";
 import "./FontIcon.scss";
 import classNames from "classnames";
 
-const FontIcon = forwardRef(({ className, onClick, iconLabel, tabIndex, icon, role = "img", ariaHidden }, iconRef) => {
-  const iconClassName = typeof icon === "function" ? "" : icon;
-  return (
-    <span
-      aria-hidden={ariaHidden}
-      className={classNames(className, "fa", iconClassName)}
-      onClick={onClick}
-      ref={iconRef}
-      aria-label={iconLabel}
-      tabIndex={tabIndex}
-      role={role}
-    >
-      {typeof icon === "function" && icon()}
-    </span>
-  );
-});
+const FontIcon = forwardRef(
+  (
+    { className, onClick, "aria-label": iconLabel, tabIndex, icon, role = "img", "aria-hidden": ariaHidden },
```

Comment:
aria-hidden, and aria-label is the same props names that IconComponent received (both components created inside the parent Icon component)